lib/main.dart

```diff
@@ -90,13 +87,26 @@ class _MyHomePageState extends State<MyHomePage> {
   @override
   void initState() {
     super.initState();
-    channel.stream.listen((message) {
-      final decodedMessage = json.decode(message);
+    try {
+      channel = WebSocketChannel.connect(
+        Uri.parse(
+            "ws://${dotenv.env["JESSE_SERVER_IP"]}:${dotenv.env["JESSE_SERVER_PORT"]}/ws"),
+      );
+
+      channel?.stream.listen((message) {
+        final decodedMessage = json.decode(message);
+        setState(() {
+          this.decodedMessage = decodedMessage;
+          _data = decodedMessage[selectedKey];
+        });
+      });
+    } catch (e) {
+      // Use mock data if the server is not available
```

Comment:
add todo: to remove it later,
at least make sure the mock data is indicate that its really mocking data, and dont ignore it in gitignore